prices/src/main/java/com/decathlon/vitamin/prices/VitaminPrices.kt

```diff
@@ -86,6 +92,8 @@ open class VitaminPriceStrikethroughLarge @JvmOverloads constructor(
     defStyleAttr: Int = R.attr.priceStrikethroughLarge
 ) : MaterialTextView(context, attrs, defStyleAttr) {
     init {
-        this.paintFlags = Paint.STRIKE_THRU_TEXT_FLAG
+        val string = SpannableString(this.text)
+        string.setSpan(StrikethroughSpan(), 0, string.length, Spanned.SPAN_EXCLUSIVE_INCLUSIVE)
+        this.text = string
```

Comment:
Maybe we can extract this traitement (and all others) in `foundation` to avoid error in the future?